File: testprojects/python_main/pipelines/PYTHON_BASIC/code/pythonbasic/test/mainone/graph/Script_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from pythonbasic.test.mainone.config.ConfigStore import *
from pythonbasic.test.mainone.udfs.UDFs import *

def Script_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    import subprocess
    result = subprocess.run(["ls"], capture_output = True, text = True)
    # result1=subprocess.run(["pwd"], capture_output=True, text=True)
    data = [StructField(item, StringType(), True) for item in result.stdout.split("\n")]
    # data.append((result1.stdout,))
    deptDF = spark.createDataFrame(data = [], schema = StructType(data))
    deptDF.printSchema()
    out0 = in0

    return out0

File: testprojects/scala_main/pipelines/SCALA_BASIC/code/src/main/scala/com/scala/main/job1/graph/Script_1.scala
package com.scala.main.job1.graph

import io.prophecy.libs._
import com.scala.main.job1.config.ConfigStore._
import com.scala.main.job1.config.Context
import com.scala.main.job1.udfs.UDFs._
import com.scala.main.job1.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Script_1 {
  def apply(context: Context, in0: DataFrame): DataFrame = {
    val spark = context.spark
    val Config = context.config
    print(s"Config: ${Config.c_test}")
    
    var out0=in0
    out0
  }

}
